pull_request_template.md

```diff
@@ -1,20 +1,18 @@
-If Applicable:
-Promotion to UAT:
-- LINK_TO_LOWER_ENVIRONMENT_PRS
+# Description
 
-----
+Please include a summary of the changes and the related issue.
 
-< list your changes here >
- 
--- Reminders (Remove this from template before submitting) --
+Fixes # (issue)
```

Comment:
Note: We don't use Github Issues, so this won't actually do anything.  Our links to JIRA use branch/PR name and commit messages.

Reply:
I was assuming we'd just link the JIRA ticket here instead, I'm even considering adding it via a templated hyperlink like

Fixes [<JIRA>](https://connectrn.atlassian.net/browse/<JIRA>#)